SUPPORT.md

```diff
@@ -0,0 +1,7 @@
+# Need help with Keycloak?
+
+Welcome to the Keycloak community. If you have some questions or need some help we are here to help, please be respectful and also remember that we simply may not have time or capacity to answer every question.
```

Comment:
I would suggest rephrasing to something like: "If you have some questions or need some help we are here to support you...", or "We are here to help you, please be respectful..."

Reply:
Updated